server/util/clickhouse/schema/schema.go

```diff
@@ -157,6 +157,10 @@ type Execution struct {
 	UserID             string
 	Worker             string
 
+	// Executor metadata
+	SelfHosted bool
+	Region     string `gorm:"type:LowCardinality(String)"`
```

Comment:
just want to confirm this gorm attributes actually works. I recalled certain gorm attributes with clickhouse don't work in the past.

Reply:
yep, double-checked in dev - the `LowCardinality` option is applied to the `effective_isolation_type` column.